src/main/msp/msp.c

```diff
@@ -1837,11 +1870,42 @@ static mspResult_e mspProcessInCommand(uint8_t cmdMSP, sbuf_t *src)
     case MSP_SET_RESET_CURR_PID:
         resetPidProfile(currentPidProfile);
         break;
-    case MSP_SET_SENSOR_ALIGNMENT:
-        gyroConfigMutable()->gyro_align = sbufReadU8(src);
-        accelerometerConfigMutable()->acc_align = sbufReadU8(src);
+    case MSP_SET_SENSOR_ALIGNMENT: {
+        // maintain backwards compatibility for API < 1.41
+        const uint8_t gyroAlignment = sbufReadU8(src);
+        sbufReadU8(src);  // discard deprecated acc_align
+#ifdef USE_MULTI_GYRO
```

Comment:
I think moving this whole conditional block down into the `} else {` of `if (sbufBytesRemaining(src) >= 3) {` will make this a bit easier to read - we do a lot of stuff here that is then immediately overwritten for newer configurator versions.